### PR TITLE
RNAshapes 3.3.0

### PR DESCRIPTION
**Pull request by mmiladi**
* [ ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by bgruening** [comment]
@mmiladi this indicates that it was compiled on a newer platform: https://travis-ci.org/bioconda/bioconda-recipes/jobs/230877072#L1465

So we need to compile it on our own.

**Comment by mmiladi** [comment]
I didn't have glib as dependency.

**Comment by mmiladi** [comment]
@bgruening What does it say? https://travis-ci.org/bioconda/bioconda-recipes/jobs/230880624#L1465
I have compiled RNAshapes_shapes_macrostate on an ubuntu 14.04

**Comment by mmiladi** [comment]
I would say there is no way to have GAPc installed without using debian packages. Chose your favourite dist from https://launchpad.net/~bibi-help/+archive/ubuntu/bibitools bellmangapc :)

**Comment by bgruening** [comment]
mh, maybe we are lucky and this is the compiler: https://github.com/conda-forge/gap-feedstock we need.

**Comment by mmiladi** [comment]
@bgruening Hilf pls :D
```
./configure --with-boost=/opt/conda/envs/rnashapes/ CPPFLAGS=-I/opt/conda/envs/rnashapes/bin/
...
checking for boostlib >= 1.36... yes
checking whether the Boost::Program_Options library is available... yes
checking for exit in -lboost_program_options... yes
checking whether the Boost::Unit_Test_Framework library is available... yes
checking for gsl-config... /opt/conda/envs/rnashapes/bin/gsl-config
checking for GSL - version >= 1.0... no
*** Could not run GSL test program, checking why...
*** The test program failed to compile or link. See the file config.log for the
*** exact error that occured. This usually means GSL was incorrectly installed
*** or that you have moved GSL since it was installed. In the latter case, you
*** may want to edit the gsl-config script: /opt/conda/envs/rnashapes/bin/gsl-config
GSL needed to compile GapC

```

**Comment by bgruening** [comment]
You need to have https://anaconda.org/conda-forge/gsl ?

**Comment by mmiladi** [comment]
Why wget in conda is not working? How can I download a file in your bioconda docker container? :-|
```
wget
bash: wget: command not found

```

**Comment by bgruening** [comment]
`conda install gnu-wget`?

**Comment by mmiladi** [comment]
Thanks! Conda-forge wget is not working!

**Comment by mmiladi** [comment]
Is it ok to have two recipes in one PR?

**Comment by mmiladi** [comment]
@bgruening  Coulld you please have a look to this?
https://travis-ci.org/bioconda/bioconda-recipes/jobs/231225280#L5464

File: recipes/gap-compiler/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+build:
+  number: 0
+  skip: True # [osx]
+package:
+  name: gap-compiler
+  version: 0.1
+requirements:
+  build: 
+  - flex
```
Comment (bgruening) [inline recipes/gap-compiler/meta.yaml]: gcc is missing here?

File: recipes/rnashapes/3.3.0/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    account of properties of the molecule''s folding space.'
+build:
+  number: 0
+  skip: True # [osx]
+package:
+  name: rnashapes
+  version: 3.3.0
+requirements:
+  build: 
+  - perl
```
Comment (bgruening) [inline recipes/rnashapes/3.3.0/meta.yaml]: gcc is missing

File: recipes/rnashapes/3.3.0/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  - gap-compiler
+  run:
+  - perl
+  - libgcc  # [linux]
+  - glib
```
Comment (bgruening) [inline recipes/rnashapes/3.3.0/meta.yaml]: glib? I don't think this is a good idea. what was the reason to add it?

**Comment by mmiladi** [comment]
@bgruening Could you please review and merge? It would be good to have both older and newer versions available in the channel

File: recipes/gap-compiler/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+about:
+  home: https://bibiserv.cebitec.uni-bielefeld.de/gapc
+  license: ''
```
Comment (bgruening) [inline recipes/gap-compiler/meta.yaml]: @mmiladi do you find an hint for a license?

**Comment by bgruening** [comment]
Looks great now! Can you figure out the license? Do we need to call them :)

**Comment by mmiladi** [comment]
Of course! :) Let me check the licence and rename gap-compiler to bellmans-gapc before merging.

**Comment by mmiladi** [comment]
@bgruening Thanks for your help! I tested the built package and it works fine. We can inform the RNAshapes and bellmans-gap teams about conda availability.

**Comment by bgruening** [comment]
Cool! Congrats! Who is updating the Galaxy wrapper?

**Comment by mmiladi** [comment]
Is there a wrapper for it :)? Ask/find a Galaxy HIWI crew ;)

**Pull request merged**